app/models/application_record.rb

```diff
@@ -1,4 +1,6 @@
 class ApplicationRecord < ActiveRecord::Base
+  require "acts_as_taggable_on"
```

Comment:
curious why this is necessary since the Gemfile is requiring it

Reply:
For some reason it was not getting loaded. Without that line the tests fails like this
```
An error occurred while loading spec_helper.
Failure/Error: acts_as_taggable

NameError:
  undefined local variable or method `acts_as_taggable' for Source (call 'Source.connection' to establish a connection):Class
# /Users/gtanzill/.rvm/gems/ruby-2.3.1@vmdb_5_7/gems/activerecord-5.1.6/lib/active_record/dynamic_matchers.rb:22:in `method_missing'
# ./app/models/source.rb:8:in `<class:Source>'
# ./app/models/source.rb:1:in `<top (required)>'
# /Users/gtanzill/.rvm/gems/ruby-2.3.1@vmdb_5_7/gems/activesupport-5.1.6/lib/active_support/dependencies.rb:292:in `require'
# /Users/gtanzill/.rvm/gems/ruby-2.3.1@vmdb_5_7/gems/activesupport-5.1.6/lib/active_support/dependencies.rb:292:in `block in require'
# /Users/gtanzill/.rvm/gems/ruby-2.3.1@vmdb_5_7/gems/activesupport-5.1.6/lib/active_support/dependencies.rb:258:in `load_dependency'
# /Users/gtanzill/.rvm/gems/ruby-2.3.1@vmdb_5_7/gems/activesupport-5.1.6/lib/active_support/dependencies.rb:292:in `require'
# ./lib/topological_inventory/persister/ar_helper.rb:12:in `block in load_environment!'
# ./lib/topological_inventory/persister/ar_helper.rb:12:in `each'
# ./lib/topological_inventory/persister/ar_helper.rb:12:in `load_environment!'
# ./spec/spec_helper.rb:24:in `<top (required)>'

Randomized with seed 49817
```